AdyenCard/Components/Card/CardAnalyticsConfiguration.swift

```diff
@@ -34,13 +33,12 @@ internal struct CardAnalyticsConfiguration: AnalyticsStringDictionaryConvertible
             self.billingAddressRequired = false
         }
         self.showsHolderNameField = configuration.showsHolderNameField
-        self.showsStorePaymentMethodField = configuration.showsStorePaymentMethodField
-        self.showsSecurityCodeField = configuration.showsSecurityCodeField
+        self.hideCVC = configuration.showsSecurityCodeField
```

Comment:
sounds like it should be the reverse?
```suggestion
        self.hideCVC = !configuration.showsSecurityCodeField
```